catalog-rest-service/src/main/resources/json/schema/tests/testCase.json

```diff
@@ -0,0 +1,117 @@
+{
+  "$id": "https://open-metadata.org/schema/tests/testCase.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "TestCase",
+  "description": "TestCase a test definition to capture data quality tests against data entities.",
+  "type": "object",
+  "javaType": "org.openmetadata.catalog.tests.TestCase",
+  "definitions": {
+    "testPlatform": {
+      "javaType": "org.openmetadata.catalog.tests.TestPlatform",
+      "description": "This schema defines the platform where tests are defined and ran.",
+      "type": "string",
+      "enum": ["OpenMetadata", "GreatExpectations", "DBT", "Deequ"]
```

Comment:
Shall we also allow `other` or `custom`? either for platforms we have not yet covered of for in-house solutions